### DISCUSSION TITLE
Disconnection of wireguard clients

### DISCUSSION THREAD

**Discussion by skntmx** [pivpn/pivpn#1458] [Bug Reports]
Hello colleagues, good day
I have pihole and pivpn wireguard in ubuntu served in the cloud, and I have mikrotik clients connected through wireguard, everything works perfectly, but it happens that when there is a power outage or internet outage in the client routers (mikrotik) they disconnect from pivpn (wireguard) you have to restart the server for them to reconnect.
I hope someone can help me to solve this conflict.
Thanks a lot!

**Comment by coolapso** [comment]
haven't actually seen that  happening ... maybe you can provide some logs from both client and server when it happens? 

this is not realy pivpn related but more (in this case) wireguard related, however maybe someone around here seen this happening and can assist you ...
